en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md

```diff
@@ -0,0 +1,57 @@
+# Connecting to Azure Service Bus
+
+This section describes how to configure WSO2 Micro Integrator to connect with Azure Service Bus.
+
+The integration between this message broker and WSO2 Micro Integrator is straightforward. The guaranteed delivery integration pattern must be used to implement the mediation flow.
+
+## Setting up the Micro Integrator with Azure Service Bus
+
+Follow the instructions below to set up and configure.
+
+1.  Download [Azure Service Bus](https://azure.microsoft.com/en-us/services/service-bus/).
```

Comment:
The Azure Service Bus is a messaging service available in the Azure Cloud. There is nothing like to download. But you have to configure it as I have explained in the document.

Reply:
Fixed.